tasks/main.yml

```diff
@@ -11,6 +11,18 @@
       paths:
         - vars
 
+- name: Check type installation of FreeIPA server
+  fail:
+    msg: "Unsupported '{{ freeipa_server_type }}' type installation of FreeIPA server"
```

Comment:
msg: "freeipa_server_type: '{{ freeipa_server_type }}' is not a valid installation type."